WalletCore/Classes/ViewModel/CashOut/CashOutBankAccountViewModel.swift

```diff
@@ -21,20 +21,38 @@ public class CashOutBankAccountViewModel {
     
     public let accountHolderAddress = Variable("")
     
+    public let accountHolderCountry = Variable("")
+
+    public let accountHolderCountryCode = Variable("")
+
+    public let accountHolderZipCode = Variable("")
+    
+    public let accountHolderCity = Variable("")
+    
     public let isValid: Observable<Bool>
     
     public init() {
-        isValid = Observable.combineLatest(
+        isValid = Observable.combineLatest([
             bankName.asObservable(),
             iban.asObservable(),
             bic.asObservable(),
-            accountHolder.asObservable()
+            accountHolder.asObservable(),
+            accountHolderAddress.asObservable(),
+            accountHolderCountry.asObservable(),
+            accountHolderCountryCode.asObservable(),
+            accountHolderZipCode.asObservable(),
+            accountHolderCity.asObservable()]
             )
             .map { return
-                $0.0.isNotEmpty &&
-                    $0.1.isNotEmpty &&
-                    $0.2.isNotEmpty &&
-                    $0.3.isNotEmpty
+                $0[0].isNotEmpty &&
```

Comment:
Please, iterate over the array instead of pointing each item individually.Functional loop would be the better approach in this case.